src/components/Header/index.jsx

```diff
@@ -25,11 +27,21 @@ class Header extends React.Component {
             <div className="col-md-auto">
               <Link to="/">Courses</Link>
             </div>
+            <Dropdown
+              title={this.props.username}
+              menuItems={[
+                <button onClick={() => apiClient.logout(process.env.LMS_BASE_URL)}>Logout</button>,
+              ]}
```

Comment:
Ideally this would look like the Studio header, since we want to look like a part of Studio. Meaning:

1. STUDIO_BASE_URL (which does not exist yet)
2. Logout -> Sign Out
3. Add Studio Home and Maintenance links (geeze those are both weird things to have there)

I'm pro-doing-these, but I'm guessing that if you checked with AJ, he might say to not bother with parts of that, yet. So might be worth your time to check in.

Reply:
Settled on:
1. Adding in STUDIO_BASE_URL (pending merge in edx-internal to work in stage/prod).
2. Changed logout to Sign Out
3. Waiting for UX to determine if we should maintain those links.

Reply:
Once we have STUDIO_BASE_URL, we should also go back and make the header logo go there too maybe